r2/r2/lib/eventcollector.py

```diff
@@ -123,6 +123,12 @@ def vote_event(self, vote, old_vote=None,
             for name, value in private_data.iteritems():
                 event.add(name, value)
 
+        if context_data:
+            referrer_url = context_data.get('referer')
+            if referrer_url:
+                event.add("referrer_url", referrer_url)
```

Comment:
If I'm thinking through this correctly, this means we'll end up with both `referrer_url` and `referer` (through being passed into `EventV2`'s constructor) being in these events, right?  Do we care about that duplication?

Alternatively, are we interested in having both `referrer_url` and `referrer_domain` available for every event (that is, by combining this block with the code down in `get_context_data()`?  I'm not sure what's desired from a data team perspective.